src/WebJobs.Script.WebHost/Diagnostics/WebHostMetricsLogger.cs

```diff
@@ -15,6 +15,7 @@ public void BeginEvent(MetricEvent metricEvent)
             if (startedEvent != null)
```

Comment:
Azure Web Apps infrastructure will consume this event for capturing aggregated metrics. We have an ETW manifest (generated using ecmangen.exe) in our infra code base, the build process generates a managed wrapper using this manifest to raise ETW events, this auto generated class uses EventProvider instead of EventSource. I don't know a way to generate an EventSource class from a given ETW manifest. EventSource class itself generates an internal manifest to raise the ETW event. Before EventSource class raises any ETW event, it registers the manifest with host (so that listeners can use this provider) and this call won't work as SDK runs in a low-privileged process.